profiles/Jonas-setup.yaml

```diff
@@ -1,15 +1,13 @@
 ---
-safe_room: 1645 # Crossing - Spooky Tree ooOoOoO
-# safe_room: 2855 # Shard - Dark Clearing
-# safe_room: 2717 # Shard - Ironwood Tree
-# safe_room: 2715 # Shard - Old Mine, Bar in Blackthorn Canyon (has rocks!)
+# safe_room: 1645 # Crossing - Spooky Tree ooOoOoO
+safe_room: 2715 # Shard - Old Mine, Bar in Blackthorn Canyon (has rocks!)
 # Hunting Settings
 
-hometown: Crossing
+hometown: Shard
 slack_username: jonas
+climbing_target: undergondola_pit
```

Comment:
I don't believe so. I'm able to train with this target, I dont know that practicing on multiple targets would help anything. I'm up for adding support for it